### DISCUSSION TITLE
Dropdown Menu Not Working After Paginate/Sort With Livewire 2

### DISCUSSION THREAD

**Discussion by reasecret** [livewire/livewire#6109] [1. Help]
I have a table for listing some items. Using Livewire 2 for pagination and sorting items. The problem is, if I refresh page (on any page number) dropdown menus are working. But if I try to sorting items or changing page, dropdown menus not working. There is no error on page or browser console. I've tried wire:ignore for dropdown menu's container div but not worked.
This is a gif for explain my issue:
![Animation](https://github.com/livewire/livewire/assets/32717963/56cfc35c-ddc8-41e8-bedd-e1a6f724913e)

Blade:
https://paste.laravel.io/af11a065-e088-48ee-8513-21982830bdc0

Component:
https://paste.laravel.io/52c187f9-0181-462c-9c74-025999cff547

Any ideas for fix this?

**Comment by SamirMaharjan** [comment]
Hello there. I am having the same issue. I just reloaded the page with url ?page=2 from js. I know this is not  the proper solution if you have the solution le letme know.

     public function updatedPage($page)
         {
               $this->emit('refreshComponent', $page);
          }
    
   

        Livewire.on('refreshComponent', function(page) 

           {
                var currentUrl = window.location.href;
                console.log(currentUrl, page);

                var url = new URL(currentUrl);
                var params = new URLSearchParams(url.search);
                var currentPage = params.get('page');

                if (currentPage !== null) {
             
                    params.set('page', page);
                } else {
                
                    params.append('page', page);
                }

                // Update the URL with the modified parameters
                url.search = params.toString();
                var newUrl = url.toString();

                // Change the location to the new URL
                window.location.href = newUrl;
          });
